mindmeld/components/entity_resolver.py

```diff
@@ -42,60 +46,284 @@
     load_index,
     resolve_es_config_for_version,
 )
+from .. import path
+from ..core import Entity
+from ..exceptions import EntityResolverConnectionError, EntityResolverError
 
 logger = logging.getLogger(__name__)
 
 
+def _is_module_available(module_name: str):
+    """
+    checks if a module is available or not (eg. _is_module_available("sentence_transformers"))
+
+    Args:
+        module_name (str): name of the model to check
+
+    Returns:
+        bool, if or not the given module exists
+    """
+    return bool(importlib.util.find_spec(module_name) is not None)
+
+
+def _get_module_or_attr(module_name: str, func_name: str = None):
```

Comment:
I think these methods are generic enough to put them in a utilities file like `_util.py`

Reply:
As of now, there is no `_util.py` in `mindmeld/components` directory. I added one to address this.